eq-author/src/constants/validationMessages.js

```diff
@@ -75,6 +75,18 @@ export const rightSideErrors = {
   },
 };
 
+export const destinationErrors = {
+  ERR_DESTINATION_MOVED: {
+    errorCode: "ERR_DESTINATION_MOVED",
+    message:
+      "Select a destination that's in the current section, or is another section",
+  },
+  ERR_DESTINATION_DELETED: {
+    errorCode: "ERR_DESTINATION_DELETED",
```

Comment:
constants

Reply:
I'm not sure if this needs to be as this is the constants file for the error messages.